.chloggen/jackgopack4_add-ocb-docker-images.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: ocb
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: create docker images for OCB
```

Comment:
Is this note accurate? Seems this PR just adds doc for the docker image

Reply:
The PR for creating the docker image is in releases repo and I couldn't figure out a good way to get it added to the changelog along with that PR

Reply:
OK, please link the PR from releases reop in the subtext

Reply:
added